samples/net/coaps_server/prj_qemu_x86.conf

```diff
@@ -14,7 +14,7 @@ CONFIG_NET_PKT_TX_COUNT=14
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=30
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
-CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=2
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
What this test needs is 4 addresses, so I expected your question and raised the limit by 1 :)